Packs/AtlassianJiraServiceManagement/Integrations/AtlassianJiraServiceManagement/AtlassianJiraServiceManagement.py

```diff
@@ -0,0 +1,831 @@
+import json
+import os
+import io
+import zipfile
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+from CommonServerUserPython import *  # noqa
+
+import urllib3
+from typing import Any
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+''' CONSTANTS '''
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'  # ISO8601 format with UTC, default in XSOAR
+ON_PREM_URL_STRUCTURE = '{}/rest/assets/1.0/'
+ON_PREM_FILE_URL_PREFIX = 'https://{}.atlassian.net/rest/servicedeskapi/attachments/'
+INTEGRATION_OUTPUTS_BASE_PATH = 'JiraAsset'
```

Comment:
why not just hardcode this everywhere? I know some integrations do this but i find it kinda hard to read and search for

Reply:
Maybe if the integration name changes in the future it would be easier to change. This product already went through name changes (used to be jira insights). You think we should change it? I personally prefer this way.